### PR TITLE
Print chroot-tools progress

### PR DESCRIPTION
**Pull request by dmcilvaney**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [ ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [ ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [ ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

Print progress of creating chroot worker, suppress spurious warning about sockets from tar.

###### Change Log  

- Progress % printout from create_worker_chroot.sh
- Ignore `tar: ./root/.gnupg/S.gpg-agent.ssh: socket ignored` etc. warnings from tar

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Local build

### CODE REVIEW

File: toolkit/tools/pkggen/worker/create_worker_chroot.sh [hunk 1]
```diff
@@ -60,6 +75,10 @@ mknod -m 600 $chroot_builder_folder/dev/console c 5 1
 mknod -m 666 $chroot_builder_folder/dev/null c 1 3
 mknod -m 444 $chroot_builder_folder/dev/urandom c 1 9
 
+# Start with filesystem, then macros, then everything else
```
Comment (cwize1) [inline toolkit/tools/pkggen/worker/create_worker_chroot.sh:78]: Why do these packages need to be installed first?

Comment (dmcilvaney) [reply toolkit/tools/pkggen/worker/create_worker_chroot.sh:78]: The filesystem is responsible for laying out the directory structure, and the majority of packages expect it to be installed already. The macros change the behavior of rpm install (by setting non-default paths etc).

Comment (dmcilvaney) [reply toolkit/tools/pkggen/worker/create_worker_chroot.sh:78]: In this case, I just want to be more defensive about ordering. I'd like to remove the requirement that the manifest file is sorted. I actually removed the parallel install stuff I was doing in the PR which necessitated installing these first, but I dropped that. I can probably just remove these lines for now.

File: toolkit/tools/pkggen/worker/create_worker_chroot.sh [hunk 2]
```diff
@@ -60,6 +75,10 @@ mknod -m 600 $chroot_builder_folder/dev/console c 5 1
...
+install_one_toolchain_rpm "$(cat "$packages" | grep "^filesystem-.*rpm$")"
+install_one_toolchain_rpm "$(cat "$packages" | grep "^mariner-rpm-macros-.*rpm$")"
+
```
Comment (dmcilvaney) [inline toolkit/tools/pkggen/worker/create_worker_chroot.sh:81]:
```suggestion
```

**Review by tobiasb-ms — APPROVED** [review]

**Pull request merged**